Server/routes/api.php

```diff
@@ -0,0 +1,26 @@
+
+<?php
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\AuthController;
+
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::post('register', [AuthController::class, 'register']);
+Route::post('login', [AuthController::class, 'login']);
+
+Route::middleware('auth:sanctum')->group(function () {
+    Route::post('/logout', [AuthController::class, 'logout']);
+    //
+    Route::get('/user', [AuthController::class, 'getUser']);
+});
```

Comment:
This is all good, but I don't see the [cors](https://laravel.com/docs/12.x/sanctum#cors-and-cookies) handler in the config, are you sure it'll work from an app opened in the browser?

If you're not familiar with cors, then I recommend [this](https://laravel.com/docs/12.x/sanctum#cors-and-cookies) to read.